In [1]:
import pandas as pd
import numpy as np

# PROYECTO EDA POR PAULA JIMÉNEZ SUAZA


## TÍTULO 📚

Análisis EDA, de la zonas más rentables para constituir una empresa de viviendas prefabricadas. 

## TEMA ✍🏻

El análisis con título anterior, se centrará en facilitarle a nuestro cliente, una empresario que quiere apostar por montar una empresa de viviendas prefabricadas, la ubicación más rentable para constiturir su empresa en España, según las Comunidades Autónomas.

Para ello, nos centraremos en cinco factores fundamentales, como son, las caraterísiticas de las viviendas prefabricadas, el incremento del precio de la vivienda, el precio por metro cuadrado, la climatología y la cantidad de suelo urbanizable disponible. 

## CLIENTE 👨🏻‍💼

Nuestro cliente es Andrés Guijarro, un trabajador del sector inmobiliario que ha decidido invertir por cuenta propia en el negocio de viviendas prefabricadas. 

Guijarro tiene dudas sobre dónde ubicar su empresa, ya que dicho factor, es decisivo a la hora de constituir su negocio. Es por ello que ha solicitado nuestros servicios, para que, en base a ciertas características como las que hemos comentado anteriormente, realicemos nuestro estudio EDA. 

## HIPÓTESIS 👀


H1. Las zonas periféricas de las ciudades tienen el metro cuadrado más económico.

H2. Las Comunidades Autónomas con el suelo urbanizable más económico son Castilla y León y La Comunidad Valenciana. 

H3. Las viviendas prefabricadas no suelen encontrarse en climas fríos, como el norte de España. 

H4. Las viviendas prefabricas suelen encontrarse en climas más cálidos.

H5. Centrarse en los nichos de mercado puede resultar ser un caso de éxito.

## ESTRUCTURACIÓN DEL PROYECTO 💻

### PASO 1. Análisis del producto 🧐
Para saber dónde nos tenemos de centrar, es importante conocer el producto de nuestro cliente. Es por ello que hemos recogido información sobre las casas prefabricadas o modulares. También se ha hecho un pequeño estudio de la competencia. 

### PASO 2. Incremento del precio de la vivienda 📈
Conocer la evolución de los precios de la vivienda es esencial para entender el contexto tanto de nuestro cliente como de su industria. Asimismo, identificar las Comunicaciones Autónomas donde se han producido aumentos más significativos en los precios será clave para orientar el análisis EDA de manera más efectiva.

Los pasos 1 y 2 son sobretodo, para conocer y entender tanto a nuestro cliente, como a su sector. Una vez hecho esto, comenzaremos con la obtención de los datos y el análisis EDA.

### PASO 3. Precio por metro cuadrado 💸
Saber el precio por metro cuadrado nos permetirá acotar la búsqueda de las Comunidades Autónomas, ya que seleccionaremos aquellas en las que la diferencia del precio del suelo entre ciudad vs municipios sea mayor. 

### PASO 4. Climatología 🌦
La comprensión de la climatología de las distintas áreas será un aspecto clave al momento de establecer la empresa, dado que el tipo de producto que ofrece nuestro cliente no puede ubicarse en cualquier región debido a las características del producto.

### PASO 5. Disponibilidad de suelo urbanizable ✅

Una vez identificadas las dos Comunidades Autónomas, el siguiente paso consistirá en determinar cuál de ellas es la más adecuada para la constitución de la empresa de nuestro cliente. Para ello, nos enfocaremos en la que ofrezca una mayor disponibilidad de suelo urbano.



## 1. ANÁLISIS DEL PRODUCTO 🧐

### 1.1 Características del producto

Debido a que nuestro cliente acaba de empezar y cuenta con una inversión no tan elevada, el tipo de producto que oferece es estándard. Tratándose de diseños de casas cuadradas o rectangulares, con cubierta plana. Fabricada de materiales como el acero y el hormigón como elemento base. Con tamaños que van desde los 40-60 metros cuadrados a los 120 metros cuadrados. Cuenta con asilamiento térmico para temperaturas poco extremas y grado de personalización de la vivienda. 
Su precio va desde los 30.000 euros (60 m²) a los 85.000 euros (120m²). El precio puede variar según las necesidades de cada cliente.

- Imágen 1. Casa prefabricada estándard

![ejemplo casa modular prefabricada](img/casa_modular.jpg)



### 1.2 Breve estudio de la competencia 


Teniendo en cuenta que nuestro cliente no sabe dónde situar su empresa, hemos realizado un búsqueda, mediante el buscador de Google Maps para ver a simple vista dónde se ubica la competencia.

Según la información aportada en el mapa, podemos observar como en la Comunidad de Madrid y en la Comunidad Valenciana hay una mayor concentración de empresas de casas prefabricadas, mientras que en el norte hay a penas unas cuentas al igual que en el sur. Como se puede observar en el mapa 1.

- Mapa 1. Empresas de casas prefabricadas en España

![mapa_espanya_empresas_casas_prefabricadas](img/competencia.png) 


### 1.3 Pequeño estudio de la competencia

Nos hemos centrado en analizar las principales características de los principales competidores, tanto en la Comunidad Valenciana, como en la Comunidad de Madrid


In [2]:
doc = ("analisis_competencia.csv")
sep = (",")

df_competencia = pd.read_csv(doc, sep=sep)
df_competencia

empresa                  ciudad comunidad  \
0                              vme                   sueca  valencia   
1                          in haus              almussafes  valencia   
2                       grupo tene                 paterna  valencia   
3                     euro casetas                 paterna  valencia   
4                    monte balsamo    la pobla de vallbona  valencia   
5                  casas carbonell                  betera  valencia   
6                              cmc                 naquera  valencia   
7                  casas cubera sl                alcorcon    madrid   
8                         bi homes      pozuelo de alarcon    madrid   
9                          maderai                  madrid    madrid   
10                    tocar madera                alcorcon    madrid   
11  casas prefabricadas steel home  las ventas de retamosa    toledo   
12                 atlantida homes                  retiro    madrid   
13                      casa redia                mostoles    madrid   
14                      rubik home             carabanchel    madrid   
15               the concrete home            torredolones    madrid   

    precio_metros_60  precio_metros_120         tipo  material  
0                  0           112251.0  minimalista  hormigon  
1                  0                NaN  minimalista  hormigon  
2               8000                0.0      rustico    madera  
3              58500           980000.0  minimalista    madera  
4                  0                0.0      rustico    madera  
5              79000           110600.0      rustico  hormigon  
6                  0           194200.0   minmalista  hormigon  
7                  0                0.0  minimalista  hormigon  
8                  0                0.0  minimalista  hormigon  
9              13500                0.0      rustico    madera  
10                 0                0.0      rustico    madera  
11                 0                0.0  minimalista  hormigon  
12                 0           226200.0  minimalista  hormigon  
13                 0                0.0  minimalista  hormigon  
14                 0                0.0  minimalista  hormigon  
15                 0                0.0  minimalista  hormigon

Del siguiente dataframe podemos obtener diversas conclusiones:
1. La competencia sigue una tendencia minimalista.
2. La mayoría de la competencia se centra en la perifería de las grandes ciudades.
3. El material utilizado para este tipo de viviendas por excelencia es el hormigón.
4. Aunque no hemos podido obtener los precios de todas las empresas, podemos ver que varía según el tipo de empresa, pero los 60 m² rondan entre los 10000 - 50000 euros y los 120m² desde los 110000 - 200000 euros. 

## 2. INCREMENTO DEL PRECIO DE LA VIVIENDA 📈

### 2.1 Análisis del IPV del 2T/2024

Según un estudio realizado por el INE, sobre el IPV (Índice de Precios de Vivienda) del segundo trimestre de 2024:
- La tasa de variación anual de IPV se situó en el 7,8%.
- La variación anual de la vivienda nueva fue del 11,2% y de segunda mano del 7,3%
- Los precios de la vivienda aumentaron en un 3,6% respecto al trimestre anterior. 

- Gráfico 1. Tasas anuales del IPV por comunidades y ciudades autónomas. Segundo trimestre 2024. En porcentaje.

![IPV_segundo_trimestre_2024](img/ipv_2t.png)



- Gráfico 2. Tasa trimestral del IPV por tipo de vivienda. Nueva y de segunda mano en porcentaje. 


![IPV_segundo_trimestre_2024](img/vivienda_ipv_2024.png)

### 2.2 Conclusiones


Centrándonos en las Comunidades Autónomas, según el gráfico 1, podemos observar que las zonas más afectadas por la subida de precios son: La Comunidad Foral de Navarra (10,3%), Aragón (9,9%), y **Andalucía (9,5%)**. 

Se observa una tendencia creciente en el IPV cada trimestre, tanto en obra nueva como en vivienda de segunda mano, como se puede observar en el gráfico 2. Este aumento en los precios sugiere que las personas buscarán alternativas a la compra tradicional de viviendas, tales como la adquisición de terrenos y la construcción de casas prefabricadas.

Según un artículo del portal de búsqueda de viviendas Fotocasa, basándose en sus estadísticas, el interés por comprar terrenos ha aumentado un 144% desde la pandemia. El 43,7% de los encuestados afirma que preferiría adquirir un terreno para construir su propia casa, mientras que el 13,7% optaría por una vivienda prefabricada. Además, se destaca que **Andalucía** es la comunidad autónoma con mayor interés en la compra o alquiler de terrenos (22%), seguida por Madrid (13%), Comunidad Valenciana (12%) y Cataluña (11%).

Entonces, ¿qué relevancia tiene el aumento de los precios de la vivienda para nuestro estudio? Este factor, junto con otros que analizaremos a continuación, debe ser considerado, ya que la subida en los precios indica una mayor demanda de clientes en determinadas áreas. Por lo tanto, las zonas con un incremento más notable en los precios de la vivienda podrían ser las más convenientes para constituir la empresa, dado el aumento de la demanda potencial.


*[Interés por comprar terreno incrementado tras la pandemia](https://blogprofesional.fotocasa.es/interes-por-comprar-terreno-incrementado-antes-pandemia/)*





## 3. PRECIO POR METRO CUADRADO DEL SUELO URBANIZABLE 💸

### 3.1 Definición

Otro factor clave a considerar al momento de ubicar la empresa es el precio por metro cuadrado del terreno. Este será un elemento determinante, ya que nuestro objetivo es establecer la empresa en aquellas áreas donde el coste del suelo sea más bajo.

Las casas modulares se fabrican en una planta y luego se transportan al terreno del cliente. Por lo tanto, el cliente inicialmente adquiere un terreno y, posteriormente, compra una vivienda modular o prefabricada.

Por ello, es crucial centrar nuestra atención en las zonas con precios de suelo más asequibles, es decir, donde se registre una mayor demanda de terrenos. 

Por ejemplo, si ubicamos la empresa en una zona donde el precio del suelo es elevado, la demanda de clientes interesados en adquirir un terreno para luego construir una casa prefabricada será baja. En estos casos, los compradores preferirán optar por una vivienda ya construida, que les resulte más rentable que comprar el terreno y luego construir una casa modular.

### 3.2 Variables a tener en cuenta y obtención de datos.

Para la obtención de los datos, hemos accedido a la página web del Ministerio de Transportes y Movilidad Sostenible, para ello nos hemos centrado en la estadística de precios de suelo urbano. 

[Estadística de precios de suelo urbano](https://apps.fomento.gob.es/BoletinOnline2/?nivel=2&orden=36000000)

Para conocer la diferencia del precio por metro cuadrado urbanizable entre ciudad y municipio, en primer lugar, hemos obtenidos los datos del precio por metro cuadrado urbanizable de las Comunidades Autónomas. Posteriormente hemos analizado los datos de los municipios de 10000 y 50000 habitantes. 

Tras hacer todos los calculos necesarios (como se especifica paso a paso en el documento 1_precio_metro_cuadrado), hemos obtenido el siguiente gráfico de barras 3:

- Gráfico 3. Comparación entre el precio medio por m² de las ciudades de las Comunidades Autónomas vs municipios de 10000 y 50000 habitantes

![diferencia_metro_cuadrado](img/newplot1.png)


### 3.3 Conclusiones

Se puede observar que el precio por metro cuadrado en las ciudades es significativamente más alto que en las áreas periféricas.

Prestando especial atención a Cantabria, notamos que en las principales ciudades el precio alcanza los 122,3 euros por metro cuadrado, mientras que en las zonas periféricas se reduce a solo 31,2 euros por metro cuadrado. Un patrón similar se encuentra en Galicia, donde el precio en las ciudades es de 112,2 euros por metro cuadrado, frente a 65,3 euros en las áreas periféricas. En Navarra, la diferencia también es notable, con un costo de 136 euros por metro cuadrado en las ciudades y 72,8 euros en la periferia.

Asimismo, en Andalucía, el precio en las ciudades es de 164,8 euros por metro cuadrado, en contraste con los 90 euros por metro cuadrado en los pueblos.

Dicha conclusión ha sido fundamental para acotar nuestra búsqueda, por lo que, para el siguiente análisis (2_climatología) nos centraremos en las dos Comunidades Autónomas anteriormente nombradas: Cantabria, Galicia, Navarra y Andalucía. 





## 4. CLIMATOLOGÍA 🌦

### 4.1 Definición

La comprensión de la climatología de las distintas áreas será un aspecto clave al momento de establecer la empresa, dado que el tipo de producto que ofrece nuestro cliente no puede ubicarse en cualquier región debido a los siguientes factores:

Tipo de techo: Los techos son de cubierta plana, lo que implica que no son aptos para soportar climas extremadamente fríos, con nieve, o lluvias intensas, ya que estas condiciones pueden comprometer su funcionalidad.

Aislamiento térmico: Aunque las viviendas cuentan con un sistema de aislamiento térmico, este está diseñado para resistir temperaturas moderadas, por lo que no es adecuado para zonas con variaciones térmicas extremas.

Con base en estas consideraciones, limitaremos nuestra búsqueda a aquellas regiones donde el clima no resulte un factor de riesgo para la vivienda. Las zonas que analizaremos serán las anteriormente nombras en el apartado 3.3

Para ello, utilizaremos los datos proporcionados por la AEMET de estas áreas para realizar un análisis detallado.

[Enlace a los productos AEMET](https://opendata.aemet.es/centrodedescargas/productosAEMET?)

### 4.2 Variables a tener en cuenta y obtención de datos

Para la obtención de los datos se ha accedido a la API de AEMET, para ello, se ha solicitado una API KEY, una vez obtenida, hemos filtrado los datos por Comunidad Autonoma (Cantabria, Galicia, Navarra y Andalucía). 

Los datos son del 2023, para así poder tener los 12 meses del año.

Se ha extraído la temperatura media máxima:

- Gráfico 4. Temperatura media máxima

![temp_max](img/newplot_2.png)

- Gráfico 5. Temperatura media mínima

![temp_min](img/newplot3.png)

- Gráfico 6. Media de las precipitaciones

![precipitaciones](img/newplot4.png)


### 4.3 Conclusiones

Hemos observado que Andalucía presenta las temperaturas más altas, lo que hace que esta región sea un área clave para centrarnos en nuestro estudio. En primer lugar, la diferencia de precio por metro cuadrado entre las zonas urbanas y el suelo urbano es casi la mitad, lo que representa una ventaja económica. Además, la climatología de la región, con bajas precipitaciones y temperaturas estables, es ideal para el tipo de producto que nuestro cliente comercializa.

No obstante, también estamos enfocando nuestra atención en las Comunidades del Norte, donde, si bien las temperaturas y precipitaciones son similares entre sí, lo que hará que elijamos una u otra en este caso, ya no va a ser la temperatura, sino la diferencia del precio por metro cuadrado. En este sentido, seleccionamos Cantabria, ya que presenta la mayor variación en el precio por metro cuadrado entre la ciudad y los municipios cercanos.

Por lo que, para el siguiente análisis (3_disponibilidad_suelo_urbanizable), seleccionamos las Comunidades de Cantabria y Andalucía










## 5. DISPONIBILIDAD DE SUELO URBANIZABLE ✅

### 5.1 Definición

### 5.2 Variables a tener en cuenta y obtención de datos

### 5.3 Conclusiones

## 6. CONTRASTACIÓN DE HIPÓTESIS 👀

Tras finalizar nuestro estudio y conocer mediante datos cuáles han sido nuestros resultados, pasaremos a contrastar las hipótesis con los datos obtenidos.


H1. Las zonas periféricas de las ciudades tienen el metro cuadrado más económico ✅
Así se afirma en el apartado 3.2 de éste documento, donde, cómo se puede observar en el gráfico 3, el precio por m² es más elevado en las ciudades que en los municipios. 

H2. Las Comunidades Autónomas con el suelo urbanizable más económico son Castilla y León y La Comunidad Valenciana. ❌
Las Comunidades Autónomas con el suelo urbanizable más económico son Castilla y León y Castilla La Mancha, así se demuestra en el gráfico 1 del documento (1_precio_metro_cuadrado). De hecho La Comunidad Valenciana es la cuarta ciudad con el precio por metro cuadrado más cara. 

H3. Las viviendas prefabricadas no suelen encontrarse en climas fríos, como el norte de España. ❌
Si bien es cierto que la mayoría de empresas de casas prefabricadas operan en La Comunidad Valenciana y La Comunidad de Madrid, cómo se observa en el mapa 1 de éste documento, en el norte de España también hay alguna empresa operando, sin embargo, no es tan común ver este tipo de viviendas en el norte debido al clima y a la concepción de viviendas modulares. 

H4. Las viviendas prefabricas suelen encontrarse en climas más cálidos. ✅
Así se puede afirmar si observamos el mapa 1 de éste documento. La mayor concentración de empresas de casas prefabricadas se sitúa en la Comunidad Valenciana, debido a su clima mediterráneo.

H5. Centrarse en los nichos de mercado puede resultar ser un caso de éxito. ✅
Tendiendo en cuenta nuestro caso de estudio y los datos que hemos obtenido, podríamos decir que montar la empresa en Andalucía sería una gran oportunidad para nuestro cliente debido a los siguientes factores:
- La Comunidad de Andalucía es de las más grandes de España.
- Cuenta con un gran número de hectáreas de suelo urbanizable. (3_disponibilidad_suelo_urbanizable)
- La diferencia de precio por metro cuadrado entre ciudad y municipio es casi de la mitad. (Gráfico 3 de éste documento)
- Nueva Ley del Suelo en Andalucía. Se simplican los procedimientos de aprobación de los diferentes instrumentos  de ordenación y con ello, una reducción de los plazos de tramitación del régimen urbanístico del suelo. Así afirma en el artículo: [Nueva Ley de Suelo en Andalucía: Impulso al Desarrollo Urbano](https://www.areall.es/nueva-ley-suelo-andalucia-impulso-desarrollo-urbano/)
- Cuenta con un clima mediterraneo que se adapta al producto de nuestro cliente. (Apartado 4.3 de éste documento)
- Poca competencia, la competencia de nuestro cliente se centra en otras zonas como la Comunidad Valenciana y la Comunidad de Madrid. (Mapa 1, de éste documento)

## 7. PRESENTACIÓN DE IDEA A NUESTRO CLIENTE 👨🏻‍💼

Tras haber realizado nuestro análisis EDA, las conclusiones a las que hemos llegado según los datos obtenidos son las siguientes:

### - Competencia 👊🏻

La competencia de nuestro cliente se centra sobretodo en La Comunidad Valenciana y La Comunidad de Madrid.

### - Precio por metro cuadrado 💸

**Andalucía** es la segunda Comunidad Autónoma con la mayor diferencia de precio por metro cuadrado entre ciudades y municipios.

### - Climatología 🌦

**Andalucía** resulta ser una zona acorde al producto de nuestro cliente debido a su clima mediterráneo y sus bajas precipitaciones. 

### - Disponibilidad de suelo urbanizable ✅

**Andalucía** cuenta con una gran disponibilidad de suelo urbanizable y esto se debe también a su amplia superfície de unas 87 599 km²

### - Normativa ⚖️

La Ley 7/2021 de Impulso para la Sostenibilidad del Territorio de Andalucía (LISTA) de **Andalucía** favorece a la obtención de terrenos, ya que se simplifican los procedimientos de aprobación y reducción de los plazos de tramitación del régimen urbanístico de suelo. 

### - **Andalucía** 🍅

Para ir un paso más allá nos gustaría presentarle a nuestro cliente un estudio que hemos encontrado de la Junta de Andalucía sobre la distribución de la superfecie de suelo según el tipo de parcela por provincia del 2022.

En el cual podemos observar que la provincia de Málaga es la que cuenta con una mayor superficie de 130262 m² (2022)de solares, mientras que en segundo lugar se encuentra Sevilla con 81554 m² (2022).

- Imagen 1. [Distribución de la superfecie del suelo según el tipo de parcela por provincias, Andalucía 2022](https://www.juntadeandalucia.es/datosabiertos/portal/dataset/clasificacion-del-grado-de-urbanizacion/resource/f59bfbc6-4a6a-4bc7-ba0d-d4982f6f51b2)

![distribucion_superficie_suelo_andalucia](img/distribución_superficie_suelo_andalucia.png)
